deployments/examples/ocis_keycloak/config/keycloak/ocis-realm.dist.json

```diff
@@ -1451,13 +1451,13 @@
       "requirement" : "ALTERNATIVE",
       "priority" : 10,
       "userSetupAllowed" : false,
-      "autheticatorFlow" : false
+      "authenticatorFlow" : false
```

Comment:
scary

Reply:
there are many tickets on this, eg. https://issues.redhat.com/browse/KEYCLOAK-16928. So this looks good to me :+1: